docs/api.rst

```diff
@@ -0,0 +1,17 @@
+Reactive API Documentation
+--------------------------
+
+
+.. toctree::
+
+    charms.reactive.decorators
+    charms.reactive.helpers
+    charms.reactive.relations
+    charms.reactive.bus
+    charms.reactive.cli
```

Comment:
There are some public API methods in there, like `set_flag`.  It's probably worth breaking them out, as you suggest.